### DISCUSSION TITLE
Looks like this is dead?

### DISCUSSION THREAD

**Discussion by rh0dium** [django-recaptcha/django-recaptcha#248] [General]
Hey there are a number of PR's outstanding.  Do  you need some help maintaining this?

Specifically support for Django 4.x and Python 3.10 would be super.  Willing to help out if needed.

**Comment by rh0dium** [comment]
Does someone have an active fork?

**Comment by Andrew-Chen-Wang** [comment]
Hi all, just published django-recaptcha429 with updated support and removal of upper Django constraint. https://github.com/Andrew-Chen-Wang/django-recaptcha429

I will not have time to make full commitment, but I will take a look at all PRs for potential merge and I'll keep package up to date with latest Django and Python versions. Cheers.

**Comment by thibaudcolas** [comment]
Hi all, **the repository has been transferred from https://github.com/praekelt/django-recaptcha to https://github.com/torchbox/django-recaptcha**. 

We got in touch with Praekelt to offer our help with maintenance, as people who were initially maintaining this package there had moved on. @tbrlpld and I are looking after the transfer (still in progress), and we’ll share more concrete plans for future maintenance once this is complete. We’re considering whether [Jazzband](https://jazzband.co/) would accept this project, among other things. @Andrew-Chen-Wang if you’d like to help maintain this project rather than your fork, please let us know.

Aside from maintenance plans, once the transfer is over our first order of business will be compatibility with recent Django releases, and continuous integration setup with GitHub Actions. There are lots of PRs already tackling those so we’ll make sure to make the most of them.

**Comment by Andrew-Chen-Wang** [comment]
@thibaudcolas I'd love to be a part-time helper/maintainer for this instead of the fork. Just send an invite. Thanks!

Edit: and yes, migrating to Jazzband would be great so long as we have enough release team members; otherwise, we run into the same situation here.

**Comment by thibaudcolas** [comment]
FYI for anyone subscribed to this – I’ve moved the conversation over to GitHub Discussions, and made a proper announcement re my comment above here: #249.
